backend/tournesol/views/rate_later.py

```diff
@@ -64,6 +64,35 @@ def perform_create(self, serializer):
         self.prefetch_entity(rate_later)
 
 
+@extend_schema_view(
+    post=extend_schema(
+        description="Add a multiple new entities to the logged user's rate-later list,"
+        " for a given poll.",
+        request=RateLaterSerializer(many=True),
+        responses={
+            201: RateLaterSerializer,
+        },
+    ),
+)
+class RateLaterBulkCreate(RateLaterQuerysetMixin, generics.CreateAPIView):
+    """
+    Create multiple rate-later entries at once.
+    Accepts an array of entities to be added to the rate-later list.
+    """
+
+    permission_classes = [IsAuthenticated]
+    serializer_class = RateLaterSerializer
+
+    def get_serializer(self, *args, **kwargs):
+        kwargs["many"] = True
+        return super().get_serializer(*args, **kwargs)
```

Comment:
A limit on the size of the bulk needs to be defined. This is to avoid timeouts, and abuse of the Youtube API (as the bulk requests could be exploited to get around the existing rate limits). 
Not sure about what a reasonable value would be? 20 ?

I suppose the extension may also need to be adapted to send batches with appropriate size.

```suggestion
    def get_serializer(self, *args, **kwargs):
        kwargs["many"] = True
        kwargs["max_length"] = RATE_LATER_BULK_MAX_SIZE
        return super().get_serializer(*args, **kwargs)
```

Reply:
Done in 2bf25b55a159ce158490d7f08912aa84d044dc0e.